src/release/breaking-changes/editable-text-scroll-into-view.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Updated EditableText scroll into view behavior
+description: >
+  Improve EditableText selection scroll into view behavior to always
+  use the current selection extent.
+---
+
+## Summary
+
+`Editable.onCaretChanged` callback is removed and with this change,
+`EditableText` behavior with regard to scrolling the selection into view
+changes.
+
+## Context
+
+To date, `EditableText` has used multiple mechanisms ensuring the selection
+extent or caret is scrolled into view on updates. 
+
+## Description of change
+
+By removing the `Editable.onCaretChanged` callback, `EditableText` will always
+use the most up to selection extent location when scrolling to show it.
+Specifically this results in improved scroll into view behavior after
+changing selection from collapsed to non-collapsed via
+`userUpdateTextEditingValue()`.
```

Comment:
What does "use the most up to selection extent location" mean?
I've made some tweaks, but would do more once I understand this phrase.

```suggestion
By removing the `Editable.onCaretChanged` callback, `EditableText` will always
use the most up to selection extent location when scrolling to show it.
Specifically, this improves scroll into view behavior after
changing selection from collapsed to non-collapsed using
`userUpdateTextEditingValue()`.
```

Reply:
The underlying issue had to do with using multiple sources of truth where one wasn't necessarily up to date (could be one frame behind).

What is clearly missing is "date" as in "up to date" :). Correcting this.